__tests__/app.test.js

```diff
@@ -13,3 +21,24 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with all topics", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body})=>{
+      expect(body.topics.length).toBe(3);
+
+      body.topics.forEach((topic) => {
+        expect(typeof topic.description).toBe("string")
+        expect(typeof topic.slug).toBe("string")
+      })
+    })
+  });
+  test("404: Error when endpoint not found" ,() => {
+    return request(app)
+    .get("/api/topic")
+    .expect(404)
+  });
+});
```

Comment:
You should also expect something coming back as a message here as well as a status code 404. And it would be worth having the misspell be more obvious, "/api/not-an-endpoint" for example.